app/actions/remote/general.ts

```diff
@@ -35,7 +35,8 @@ async function getDeviceIdForPing(serverUrl: string, checkDeviceId: boolean) {
     return getDeviceToken(appDatabase);
 }
 
-export const doPing = async (serverUrl: string, verifyPushProxy: boolean) => {
+// Default timeout interval for ping is 5 seconds
+export const doPing = async (serverUrl: string, verifyPushProxy: boolean, timeoutInterval = 5) => {
```

Comment:
On the title of the PR you say 3 seconds, but here is 5 seconds.

Reply:
oh lol... 😂 

what you think, should we use 3 or 5?

Reply:
I would go with 5, but 0/5 :sweat_smile:

Reply:
done!